snuba/utils/streams/backends/kafka.py

```diff
@@ -650,13 +651,22 @@ def closed(self) -> bool:
 
 
 def get_default_kafka_configuration(
-    storage_key: Optional[StorageKey] = None,
+    storage_key: Optional[
+        StorageKey
+    ] = None,  # TODO: To be removed once STORAGE_BROKER_CONFIG and DEFUALT_STORAGE_BROKERS are gone
+    topic: Optional[KafkaTopic] = None,
     bootstrap_servers: Optional[Sequence[str]] = None,
     override_params: Optional[Mapping[str, Any]] = None,
 ) -> KafkaBrokerConfig:
     default_bootstrap_servers = None
     default_config: Mapping[str, Any]
-    if storage_key is not None:
+
+    if storage_key is None:
+        assert topic is None
+    else:
+        assert topic is not None
+
+    if storage_key is not None and topic is not None:
```

Comment:
if storage_key is not None topic cannot be None as per line 667.

Reply:
I know, just need to keep mypy happy